filament/backend/test/test_ComputeBasic.cpp

```diff
@@ -22,25 +22,29 @@ using namespace filament::backend;
 TEST_F(ComputeTest, basic) {
     auto& driver = getDriverApi();
 
-    std::string_view shader_gles310 = {R"(
+    std::string shader_gles310 = {R"(
```

Comment:
why the change to `string` from `string_view`?

Reply:
The shaders passed to backends need to have a null terminator, which `std::string_view` doesn't guarantee.